File: package/go/src/fake_chinese_checker.go
package src

// IsFakeChinese checks if a string contains only Chinese characters (Kanji)
func IsFakeChinese(text string) string {
	if text == "" {
		return "誤"
	}

	for _, char := range text {
		// Check if character is a CJK ideograph (Chinese/Japanese/Korean unified ideographs)
		// Unicode range U+4E00 to U+9FFF
		if char < '\u4e00' || char > '\u9fff' {
			return "誤"
		}
	}

	return "正"
}

File: package/java/src/main/java/com/programknock/FakeChineseChecker.java
package com.programknock;

public class FakeChineseChecker {

    public static String isFakeChinese(String text) {
        if (text == null || text.isEmpty()) {
            return "誤";
        }

        for (char c : text.toCharArray()) {
            if (c < '\u4e00' || c > '\u9fff') {
                return "誤";
            }
        }

        return "正";
    }
}
